editor/modes/visual-editor/block-list.js

```diff
@@ -248,18 +268,24 @@ class VisualEditorBlockList extends Component {
 						onKeyDown={ noop }
 					/>
 				}
-				<div className="editor-visual-editor__continue-writing">
+				<div
+					className={ continueWritingClassname }
+					onFocus={ this.onContinueWritingFocus }
+					onBlur={ this.onContinueWritingBlur }
+				>
 					<Inserter position="top right" />
 					<button
```

Comment:
Tried that, and seems the IconButton uses some default focus style that maybe is not appropriate here. Instead, it should maybe use the same focus style of the items in the inserter menu?

<img width="415" alt="screen shot 2017-07-18 at 23 22 55" src="https://user-images.githubusercontent.com/1682452/28340484-43eb6bbe-6c10-11e7-853f-9436e6882376.png">